doc/classes/OS.xml

```diff
@@ -325,7 +325,7 @@
 			</description>
 		</method>
 		<method name="get_unix_time" qualifiers="const">
-			<return type="int">
+			<return type="float">
```

Comment:
Can you improve the documentation while at it? It should at least specify the unit used.